package.json

```diff
@@ -32,6 +32,9 @@
     "@angular/http": "^2.2.0",
     "@angular/platform-browser": "^2.2.0",
     "core-js": "^2.4.1",
+    "firebase": "^3.6.7",
+    "google-cloud": "^0.45.1",
+    "image-diff": "^1.6.3",
```

Comment:
Removed `firebase` dependency. Old version of `firebase-admin` have wrong types for database.